README.md

```diff
@@ -1,2 +1,70 @@
 # Telnyx Video SDK for iOS
 This repository contains releases of Telnyx's Video SDK for iOS. These releases can be installed as a CocoaPod or manually into your project.
```

Comment:
@guillermobattistel I think we can say something like you mentioned on line 5, here:
"Currently a release is installed through CocoaPods."

Reply:
Done